src/e2e/specs/landing.spec.ts

```diff
@@ -40,15 +40,17 @@ test.describe(`${process.env.E2E_TEST_ENV} - Landingpage element verification`,
     );
   });
 
-  test("Verify landingpage elements", async ({ landingPage }) => {
+  // skipping for now, as testrail for landing page needs to be updated
+  test.skip("Verify landingpage elements", async ({ landingPage }) => {
     // confirm landing page elements are visible
     await expect(landingPage.whyUseMonitorSec).toBeVisible();
     await expect(landingPage.howItWorksSec).toBeVisible();
     await expect(landingPage.questionsAboutSec).toBeVisible();
     await expect(landingPage.seeIfDataBreachSec).toBeVisible();
   });
 
-  test("Verify that the site footer is displayed correctly @ui", async ({
+  // same as above
```

Comment:
Just because people (or at least: I do...) tend to insert new tests in between other tests, it might be safer to just copy-paste the comment from above. (Likewise below.) Also avoids people needing to scroll up.